.gitignore

```diff
@@ -2,33 +2,353 @@
 /build
 AssemblyInfo.cs
 
-# Standard VS.NET and ReSharper Foo
-.vs/*
-src/*/obj
-src/*/bin
-samples/*/obj
-samples/*/bin
-*.csproj.user
-*ReSharper.user
-_ReSharper*
-*.suo
 *.cache
 * Thumbs.db
 
 # Leftovers from merging
 *.orig
 *.bak
```

Comment:
We can probably get rid of these 2 as well. I've never seen a merge leave these behind unless you abort it wrong.